common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -53,19 +58,40 @@ public void onTitleScreenInit(TitleScreenInitEvent.Post e) {
     public void onTitleScreenInit(ScreenInitEvent e) {
         if (!(e.getScreen() instanceof TitleScreen titleScreen)) return;
 
-        addWynncraftButton(titleScreen);
+        if (!initBootPassed && autoConnect.get()) {
+            autoConnect();
+        } else {
+            addWynncraftButton(titleScreen);
```

Comment:
Why is this an else? I presume that you will want the button added even if you autoconnect, so it is there after you log out from Wynncraft.

Reply:
the button is added with **onTitleScreenInit(TitleScreenInitEvent.Post e)** which triggers after you log out from Wynncraft or go back to the titlescreen from any other menu.
the addWynncraftButton() is only needed in the **onTitleScreenInit(ScreenInitEvent e)** to render the button on the "initial start"
therefore if you want to autoconn on the "initial start" you don't necessarily need it